src/ResourceManager/Resources/Commands.ResourceManager/Cmdlets/Implementation/ResourceGroupDeployments/NewAzureResourceGroupDeploymentCmdlet.cs

```diff
@@ -45,6 +47,9 @@ public class NewAzureResourceGroupDeploymentCmdlet : ResourceWithParameterCmdlet
         [ValidateSet("RequestContent", "ResponseContent", "All", "None", IgnoreCase = true)]
         public string DeploymentDebugLogLevel { get; set; }
 
+        [Parameter(Mandatory = false, ValueFromPipelineByPropertyName = true, HelpMessage = "The name of a deployment to roll back to on error, or use as a flag to roll back to the last successful deployment.")]
```

Comment:
Please regenerate all the help files to reflect this change: https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md